src/util.h

```diff
@@ -451,7 +451,7 @@ struct MallocedBuffer {
 
   MallocedBuffer() : data(nullptr) {}
   explicit MallocedBuffer(size_t size) : data(Malloc<T>(size)), size(size) {}
-  MallocedBuffer(char* data, size_t size) : data(data), size(size) {}
+  MallocedBuffer(T* data, size_t size) : data(data), size(size) {}
   MallocedBuffer(MallocedBuffer&& other) : data(other.data), size(other.size) {
```

Comment:
how does this fare with hard-coded `char` type in `Malloc`? shouldn't we `generalize` Malloc as well?
https://github.com/nodejs/node/blob/54ce229d309d6dc2991f434a1210ee4117963faf/src/util-inl.h#L396

Reply:
`Malloc` is already a template -- the template-less variant you quoted just calls the templated one :)

Reply:
It is templated:
https://github.com/nodejs/node/blob/54ce229d309d6dc2991f434a1210ee4117963faf/src/util-inl.h#L381-L386
L396 is just a specialization, and shortcut.

Reply:
ok, I see; thanks!